planning/autoware_manual_lane_change_handler/README.md

```diff
@@ -0,0 +1,76 @@
+# Manual Lane Change Handler
+
+## Purpose
+
+`Manual Lane Change Handler` helps shift the preferred-primitives of a route to the left or right.
+
+### Services
```

Comment:
There are server and client for the ndoe please make sure that you have both of them described.